components/automate-cli/pkg/verifyserver/models/populatewith_test.go

```diff
@@ -0,0 +1,432 @@
+							AdminCert: "-----BEGIN CERTIFICATE-----\nMIIDgTCCAmmgAwIBAgIJAPMNo6eG0UBZMA0GCSqGSIb3DQEBCwUAMGMxCzAJBgNV\nBAYTAlVTMRMwEQYDVQQIDApXYXNoaW5ndG9uMRAwDgYDVQQHDAdTZWF0dGxlMRow\nGAYDVQQKDBFDaGVmIFNvZnR3YXJlIEluYzERMA8GA1UEAwwIcHJvZ3Jlc3MwHhcN\nMjMwNTIyMTEzOTE5WhcNMjYwNTIxMTEzOTE5WjBkMQswCQYDVQQGEwJVUzETMBEG\nA1UECAwKV2FzaGluZ3RvbjEQMA4GA1UEBwwHU2VhdHRsZTEaMBgGA1UECgwRQ2hl\nZiBTb2Z0d2FyZSBJbmMxEjAQBgNVBAMMCWNoZWZhZG1pbjCCASIwDQYJKoZIhvcN\nAQEBBQADggEPADCCAQoCggEBAPRoFiZnKE2OxuLXrJu/FhIFisgpN0KtnaD8V4Cd\n/FVjj/zknopa+9+hYUOqnScBKGWZBvsV7kZC43UgxMNukKBjEBVfoqaToTqIpyam\nSjmnkra8SjYDT/RX2+HZbmZ0mOMY62MOC1gBhH1oMx0t3JqAp9vcCdo7H5Oi1C6K\nPScXJhpMobAruyHn/L0/K3R+Wh4HRqmd2IluH56UPMba3xAsGK+y4iXSkjnXQ9lJ\nVVxYgGQnZFUA/LLyDL/A4iewTTyjb7RKsjHPZ/uP7hkUp9cRSIGimLDLuTloV6ra\nBcXav812mL3q42w/gJfEeRR37uNwVlBUc+Jc0hc/owItstcCAwEAAaM3MDUwHQYD\nVR0lBBYwFAYIKwYBBQUHAwIGCCsGAQUFBwMBMBQGA1UdEQQNMAuCCWNoZWZhZG1p\nbjANBgkqhkiG9w0BAQsFAAOCAQEAD4uod0aFfmwHuXshXFPcvu5gkkvd2Prd8pMz\nTiOu0A1IYgVHvAg0Bqs8bReZLsijUKhhjmlfwaUrW1i2r5jYp9Acd2K/rKwl7rov\ngaP60IgkszFCjfZFU1Zb0+6OlPuG8PquMPSJuJDFm5mQkw52vb81guvF/cGlKCuI\nfPwEr1CSOyHmQMQLLJsdzfB2RQ6MfG1pCH1FNPCfWjK8Rrd0Hic/sQ4Yd6q/Bmek\nb9KA5lUyk1Ox/YBBfm4RfmeKRqQKtASF8UJG3eCut0h3+uqyQpCxAx/8MlipkF7g\n1EvU3J0+e99VMZJ/wNlrAla08VsV/hjXd+NIZYkl31cghLXmTA==\n-----END CERTIFICATE-----",
+						},
+					},
+				},
+				ExternalOS: ExternalOS{
+					OSDomainName:   "",
+					OSDomainURL:    "",
+					OSUsername:     "",
+					OSUserPassword: "",
+					OSCert:         "",
+					OSRoleArn:      "",
+				},
+				ExternalPG: ExternalPG{
+					PGInstanceURL:       "",
+					PGSuperuserName:     "",
+					PGSuperuserPassword: "",
+					PGDbUserName:        "",
+					PGDbUserPassword:    "",
+					PGRootCert:          "",
+				},
+				DeploymentState: "",
+				APIToken:        "",
+			},
+			wantErr: false,
+			err:     nil,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			haConfig := &config.HaDeployConfig{}
+			err := haConfig.Parse(tt.filePath)
+			if err != nil {
+				t.Errorf("Error parsing HaDeployConfig: %v", err)
+				return
+			}
```

Comment:
We can just do assert.NoError(t, err, Error parsing HaDeployConfig)

Reply:
Okey, used assert.NoErrorf(t, err, "Error parsing HaDeployConfig: %v", err) since i want to format error.